test/standalone/Vegetation/canopy_model.jl

```diff
@@ -190,7 +190,7 @@ for FT in (Float32, Float64)
             @test getproperty(prognostic_types(canopy), component) ==
                   prognostic_types(getproperty(canopy, component))
         end
-        Y.canopy.hydraulics[1] = plant_ν
+        parent(Y.canopy.hydraulics.ϑ_l) .= plant_ν
```

Comment:
do we need `Array(parent(...))` here?

Reply:
No, because we want to change a value on the GPU. If we had `Array`, we would change a value of a copy on the CPU, which is not the intent